assets/section-main-product.css

```diff
@@ -343,6 +347,10 @@ a.product__text {
 }
 
 @media screen and (min-width: 750px) {
+  .product__info-container {
+    max-width: 60rem;
```

Comment:
this can make it feel like the content is left aligned and not taking the rest of space that other sections use: [video](https://screenshot.click/22-01-opirh-79o3e.mp4)

I don't think it's an issue to have the content take the available space 🤔  plus you can restrict it if you want to by reduce the layout width or change the amount of space the media take.

Reply:
Alrighty, that makes sense. Thanks for the resources Meli 👍